db/controllers/reviews.controller.js

```diff
@@ -2,11 +2,15 @@ const { fetchReview } = require("../models/reviews.model");
 
 exports.getReview = (request, response, next) => {
   const reviewToFetch = request.params;
-  fetchReview(reviewToFetch)
-    .then((fetchedReview) => {
-      response.status(200).send(fetchedReview);
-    })
-    .catch((err) => {
-      next(err);
-    });
+  if (reviewToFetch.review_id.match(/([0-9])+/)) {
```

Comment:
you don't need to use a match here. PSQL has a very specific error code when handling an invalid input. This is covered in the error handling portion of your notes from last week.